src/Reader/readpar.f90

```diff
@@ -1032,6 +1042,23 @@ SUBROUTINE readdr(IO, EQN, DISC, BND, IC)
              DISC%DynRup%RS_iniSlipRate1 = RS_iniSlipRate1! V_ini1, initial sliding velocity
              DISC%DynRup%RS_iniSlipRate2 = RS_iniSlipRate2! V_ini2, initial sliding velocity
              DISC%DynRup%t_0      = t_0       ! forced rupture decay time
+             DISC%DynRup%ThermalPress = thermalPress !switches TP on (1) or off(0)
+             IF (DISC%DynRup%ThermalPress.EQ.1) THEN !additional parameters
+                 logInfo0(*) 'Thermal pressurization assumed'
+                 !physical
+                 DISC%DynRup%TP_grid_half_width = TP_grid_half_width
```

Comment:
what is the difference between TP_grid_half_width and TP_grid_space_distance?

Reply:
ok maybe a bit misleading. I can change TP_grid_half_width into TP_half_width_shear_zone

Reply:
no TP_grid_space_distance was formely TP_dlDwn which means it is the logarithmic space distance of the grid in the z-direction.

Reply:
it has nothing to do with half_width and the space distance is logarithmic. Ok, I could do TP_grid_log_space_distance.